src/legacy/core_plugins/interpreter/index.ts

```diff
@@ -28,7 +28,7 @@ export default function InterpreterPlugin(kibana: any) {
     require: ['kibana', 'elasticsearch'],
     publicDir: resolve(__dirname, 'public'),
     uiExports: {
-      interpreter: ['plugins/interpreter/load_legacy_server_function_wrappers'],
+      interpreter: ['plugins/interpreter/canvas/load_legacy_server_function_wrappers'],
```

Comment:
should this be moved into canvas plugin?

Reply:
Yes, we have a separate ticket for that: https://github.com/elastic/kibana/issues/44326